examples/carbon-relay-ng-kafka.ini

```diff
@@ -0,0 +1,120 @@
+## Global settings ##
```

Comment:
I would remove everything that is not vital in this, along with comments
It's a specific example conf, not the main example

This will avoid us to have to leave wrong files if we modify unrelated config